src/e-pr.js

```diff
@@ -50,8 +65,9 @@ try {
 
 program
   .description('Open a GitHub URL where you can PR your changes')
+  .option('-r, --repo <repository>', 'Repository to use', defaultRepo)
   .option('-s, --source <source_branch>', 'Where the changes are coming from', defaultSource)
   .option('-t, --target <target_branch>', 'Where the changes are going to', defaultTarget)
   .parse(process.argv);
 
-open(`https://github.com/electron/electron/compare/${program.target}...${program.source}?expand=1`);
+open(`https://github.com/${program.repo}/compare/${program.target}...${program.source}?expand=1`);
```

Comment:
This isn't right, we want to PR to be electron/electron but the "source" branch to be from the fork